ast/src/analyzed/mod.rs

```diff
@@ -337,13 +337,9 @@ impl<T> Analyzed<T> {
                     }
                 };
                 let row_offset = public_declaration.index as usize;
-                (column_name, column_idx as usize, row_offset)
+                (public_id, (public_name, column_idx as usize, row_offset))
             })
-            .collect::<Vec<_>>();
-
-        // Sort, so that the order is deterministic
-        publics.sort();
```

Comment:
No longer necessary, because a `BTreeMap` is always sorted by ID (in this case the ID).